include/psa/crypto_values.h

```diff
@@ -787,6 +803,13 @@
 #define PSA_ALG_SHA3_384                        ((psa_algorithm_t)0x02000012)
 /** SHA3-512 */
 #define PSA_ALG_SHA3_512                        ((psa_algorithm_t)0x02000013)
+/** The first 64 bytes of the SHAKE256 output.
```

Comment:
There is a difference in how SHAKE256 is defined in FIPS202 and in RFC8032: in the former this algorithm is parameterized by the bit-size of the output length, the definition used in RFC8032 is parameterized by the octet-length of the output.

Thus `PSA_ALG_SHAKE256_64` maps easily onto the normative Ed448ph definition, but is not consistent with the FIPS202 normative definition for SHA3 and SHAKE functions, nor with the use of a bit-length suffix for the other SHA hash algorithms.

Would `PSA_ALG_SHAKE256_512` be more appropriate for PSA Crypto?.

Reply:
Good point. I have no strong opinion here. It could even be `PSA_ALG_ED448_PREHASH`!

Reply:
I see. That would inspire some confusion with the proposed `PSA_ALG_ED448PH` signature algorithm - so it might be better to name it after the underlying hash algorithm rather than the purpose to which it is put?

Reply:
I actually think naming the hash for its intended purpose would make sense, but I don't really think `PSA_ALG_ED448_PREHASH` is a good name for this reason.